src/main/webapp/app/core/legal/imprint.module.ts

```diff
@@ -4,10 +4,10 @@ import { ArtemisSharedModule } from 'app/shared/shared.module';
 
 import { ImprintRoutingModule } from 'app/core/legal/imprint-routing.module';
 import { ImprintComponent } from 'app/core/legal/imprint.component';
+import { ArtemisMarkdownModule } from 'app/shared/markdown.module';
 
 @NgModule({
     declarations: [ImprintComponent],
-    imports: [CommonModule, ArtemisSharedModule, ImprintRoutingModule],
-    exports: [ImprintComponent],
```

Comment:
Why is this export not needed anymore?

Reply:
I cannot think of a scenario where I want to reuse the imprint component in a different place. --> removed for now.
In case one want to use it later in another place, it's no big deal to add the export again